bears/general/CasingBear.py

```diff
@@ -0,0 +1,153 @@
+import re
+
+from coalib.bears.LocalBear import LocalBear
+from coalib.bearlib.naming_conventions import (
+    to_camelcase, to_pascalcase, to_snakecase)
```

Comment:
This was the standard I remember seeing in coala when I started - been using it for some time now :P